packages/mariadb/package.json

```diff
@@ -36,10 +36,10 @@
     "@sequelize/core": "workspace:*",
     "@sequelize/utils": "workspace:*",
     "dayjs": "^1.11.10",
-    "lodash": "4.17.21",
-    "mariadb": "3.3.0",
-    "semver": "7.6.0",
-    "wkx": "0.5.0"
+    "lodash": "^4.17.21",
+    "mariadb": "^3.3.0",
+    "semver": "^7.6.0",
+    "wkx": "^0.5.0"
   },
   "devDependencies": {
```

Comment:
These devDependencies were added in error and can be removed.

Reply:
Indeed, because they do not run their own unit tests. Removed from the applicable packages